ooniprobe/View/TestResults/Footer/UploadFooterViewController.m

```diff
@@ -2,6 +2,9 @@
 #import "MessageUtility.h"
 #import <QuartzCore/QuartzCore.h>
 #import "SettingsUtility.h"
+#import <mkall/MKCollector.h>
+#import "TestUtility.h"
+#import "MBProgressHUD.h"
```

Comment:
FWIW, I suggest to organise headers as follows:

1. include the header of the class that you are implementing

2. include all system headers

3. include all dependency headers

4. include all other headers that belong to this project

Reply:
reodering dependency (and move them better between .h and .m files) is a thing I wanted to do soon, I will have to study more obj-c before doing it.